osgi.specs/docbook/cmpn/001/introduction.xml

```diff
@@ -662,6 +662,17 @@
               linkend="service.log.stream"/></code></entry>
             </row>
 
+            <row>
+              <entry><xref linkend="service.features"
```

Comment:
may use singular `feature` not `features`

Reply:
I think singular would then map to 'Feature Service Specification' which I think is a bit too general and potentially confusing. I would like to leave the name at 'Features Service Specification' which would then map to `service.features`.